src/stripeActions/process-data/index.js

```diff
@@ -174,99 +77,182 @@ export const processData = async (payload) => {
+                      currency: "usd",
+                      metadata: !plans[i][index].planIncludes
+                        ? {}
+                        : plans[i][index].planIncludes,
+                      unit_amount: plans[i][index].price * 100,
+                    },
                   };
-                }
-
-                //  Structure Requirement for /create-price API (recurring)
-                const pricePayload = {
-                  sanityToken,
-                  hashKey: plans[i].hashKey,
-                  stripeSecretKey: plans[i].stripeSKey,
-                  apiVersion: !plans[i].apiVersion
-                    ? "2020-08-27"
-                    : plans[i].apiVersion,
-                  productId: responseCreateProduct.data.id,
-                  isRecurring: true,
-                  monthly_unit_amount: parseInt(plans[i][index].monthlyPrice),
-                  yearly_unit_amount: parseInt(plans[i][index].yearlyPrice),
-                  currency: "usd",
-                };
-
-                // Creating Price
-                const responseCreatePrice = await createPrice(pricePayload);
-                if (responseCreatePrice.meta.status !== 200) {
-                  return {
-                    status: 500,
-                    statusText: `Product Not Created`,
+                  try {
+                    const response = await fetch(createPriceURL, {
+                      headers: {
+                        "Content-Type": "application/json",
+                      },
+                      method: "POST",
+                      body: JSON.stringify(createPricePayload_VariantB),
+                    });
+                    await response.json();
+                  } catch (error) {}
+                } else {
+                  const createPricePayload_VariantAC_Monthly = {
+                    credentials,
+                    StripeParams: {
+                      product: createProduct.id,
+                      currency: "usd",
+                      metadata: !plans[i][index].planIncludes
+                        ? {}
+                        : plans[i][index].planIncludes,
+                      unit_amount: plans[i][index].monthlyPrice * 100,
+                      recurring: {
+                        interval: "month",
+                      },
+                    },
                   };
+                  const createPricePayload_Variant_AC_Yearly = {
+                    credentials,
+                    StripeParams: {
+                      product: createProduct.id,
+                      currency: "usd",
+                      metadata: !plans[i][index].planIncludes
+                        ? {}
+                        : plans[i][index].planIncludes,
+                      unit_amount: plans[i][index].yearlyPrice * 100,
+                      recurring: {
+                        interval: "year",
+                      },
+                    },
+                  };
```

Comment:
This payload needs to be updated as per https://github.com/webriq-pagebuilder/app/pull/63#discussion_r719403192